src/ccpa/sourcepoint.ts

```diff
@@ -48,16 +51,6 @@ export const init = (pubData = {}) => {
 				onMessageReceiveData: (data) => {
 					resolveWillShowPrivacyMessage?.(data.msg_id !== 0);
 				},
-				onMessageChoiceSelect: (_choiceId, choiceTypeID) => {
```

Comment:
Why is this being removed entirely?

Reply:
it's fired when a user clicks on the privacy message (dismissed it without making a choice etc). since we're running callbacks as soon as the `uspapi` is available we don't need to handle that case now. i suspect this may be true for TCF too, but we can look at that after